### PR TITLE
Fix kedro-starter templates alias list

### PR DESCRIPTION
**Pull request by noklam**
Signed-off-by: noklam <nok.lam.chan@quantumblack.com>

## Description

Fix https://github.com/kedro-org/kedro-starters/issues/84

The list of aliases was not updated with the change `mini-kedro` -> `standalone-datacatalog` renaming.

## Development notes

Updated the aliases list

## Checklist

- [ ] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1446"></a>

### CODE REVIEW

File: kedro/framework/cli/starters.py [hunk 1]
```diff
@@ -280,6 +280,7 @@ def _get_cookiecutter_dir(
             no_input=True,
             directory=directory,
         )
+        print("****", template_path, cookiecutter_dir)
```
Comment (AntonyMilneQB) [inline kedro/framework/cli/starters.py:283]: I guess this isn't meant to be here any more...

**Review by AntonyMilneQB — APPROVED** [review]
Thanks for fixing this!

File: kedro/framework/cli/starters.py (hunk 1 above)
Comment (noklam) [reply kedro/framework/cli/starters.py:283]: Indeed 😅

**Comment by noklam** [comment]
@MerelTheisenQB Needs your help to approve and merge this fix since the delta table test is failing, thank you!

**Review by MerelTheisenQB — APPROVED** [review]
👍

**Pull request merged**